twingate/internal/provider/resource/resource.go

```diff
@@ -497,13 +498,24 @@ func convertResource(data *schema.ResourceData) (*model.Resource, error) {
 	}
 
 	isBrowserShortcutEnabled, ok := data.GetOkExists(attr.IsBrowserShortcutEnabled) //nolint
-	if val := isBrowserShortcutEnabled.(bool); ok {
+	if val := isBrowserShortcutEnabled.(bool); ok && isAttrKnown(data, attr.IsBrowserShortcutEnabled) {
 		res.IsBrowserShortcutEnabled = &val
 	}
 
+	if res.IsBrowserShortcutEnabled != nil && *res.IsBrowserShortcutEnabled && strings.HasPrefix(res.Address, "*.") {
```

Comment:
Wildcard addresses are more than just the address being prefixed with `*.`. Wildcard addresses include:
- An address that has a wildcard, which is `*` and `?`, e.g. `redis-?-blah.internal` or `redis-*-blah.internal`
- CIDR ranges, e.g. `10.0.0.0/16`

Reply:
added test cases